<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XPlor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent...</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentOpenRecent"/>
     </property>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import...</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::FolderNew"/>
     </property>
    </widget>
    <addaction name="actionNew_File_2"/>
    <addaction name="actionNew_Fast_File"/>
    <addaction name="actionNew_Zone_File"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Fast_File"/>
    <addaction name="actionOpen_Zone_File"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuRecent"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuUndo_History">
     <property name="title">
      <string>Undo History...</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuRedo_History">
     <property name="title">
      <string>Redo History...</string>
     </property>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionRename"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind_2"/>
    <addaction name="actionEdit_Value"/>
    <addaction name="actionEdit_as_Hex"/>
    <addaction name="separator"/>
    <addaction name="menuUndo_History"/>
    <addaction name="menuRedo_History"/>
    <addaction name="actionClear_Undo_History"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_File_2">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_Fast_File">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>New Fast File</string>
   </property>
  </action>
  <action name="actionNew_Zone_File">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>New Zone File</string>
   </property>
  </action>
  <action name="actionOpen_Fast_File">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open Fast File</string>
   </property>
  </action>
  <action name="actionOpen_Zone_File">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open Zone File</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actione">
   <property name="text">
    <string>e</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionNew_Fast_File_2">
   <property name="text">
    <string>New Fast File</string>
   </property>
  </action>
  <action name="actionNew_Zone_File_2">
   <property name="text">
    <string>New Zone File</string>
   </property>
  </action>
  <action name="actionFrom_Clipboard">
   <property name="text">
    <string>From Clipboard</string>
   </property>
  </action>
  <action name="actionMaterial">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut"/>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionEdit_Value">
   <property name="text">
    <string>Edit Value</string>
   </property>
  </action>
  <action name="actionEdit_as_Hex">
   <property name="text">
    <string>Edit as Hex</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>d</string>
   </property>
  </action>
  <action name="actiond_2">
   <property name="text">
    <string>d</string>
   </property>
  </action>
  <action name="actionClear_Undo_History">
   <property name="text">
    <string>Clear Undo History</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionChange_Icons">
   <property name="text">
    <string>Change Icons</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
   </property>
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionFind_2">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
